### ISSUE TITLE
Remove conda_build_config.yaml

### ISSUE THREAD

**Issue by isuruf** [conda-forge/mpi4py-fft-feedstock#6]
Since the python module only depends on mpi4py at runtime, this is not needed. Rerendering should reduce the number of jobs to 3 per OS

**Comment by mikaem** [comment]
Yes, I was actually wondering about that myself. I'll make a new PR.

**Comment by mikaem** [comment]
Fixed by PR #7
